File: 백준/Bronze/10833. 사과/사과.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
      int T, N, A, count=0;
      Scanner s = new Scanner(System.in);
      T = s.nextInt();
      for(int i=0; i < T; i++){
        N = s.nextInt();
        A = s.nextInt();
        if (N <= A){
            count += A % N;
        }else{
            count += A;
        }
    }
      System.out.print(count);
    }
}

File: 백준/Bronze/10833. 사과/사과.cc
#include<stdio.h>

int main() {
    int T, N, A, count = 0;
    scanf("%d", &T);
    for(int i=0; i < T; i++){
        scanf("%d %d", &N, &A);
        if (N <= A){
            count += A % N;
        }else{
            count += A;
        }
    }
    printf("%d", count);
}